Makefile

```diff
@@ -63,7 +62,7 @@ generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Build the docker image
-docker-build: test
+docker-build:
```

Comment:
Why are we removing test target?

Reply:
Oops, sorry removed that while testing locally. Test target is back